internal/kafka/internal/presenters/cluster.go

```diff
@@ -0,0 +1,56 @@
+package presenters
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/api/public"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/services"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/client/observatorium"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
+)
+
+func PresentEnterpriseCluster(cluster api.Cluster, observabilityConfiguration *observatorium.ObservabilityConfiguration, fleetShardParams services.ParameterList) (public.EnterpriseCluster, *errors.ServiceError) {
+	observabilityParams := []public.ObservabilityParameter{
```

Comment:
@machi1990 @pawelpaszki if we're going to install the Observability Operator through SyncSets even for Enterprise, we probably don't need to return this.

Reply:
I agree.

Reply:
removed